test/testcafe/spec/TerminalReturnEmail_spec.js

```diff
@@ -0,0 +1,28 @@
+import IdentityPageObject from '../framework/page-objects/IdentityPageObject';
+import TerminalPageObject from '../framework/page-objects/TerminalPageObject';
+import { RequestMock } from 'testcafe';
+import terminalReturnEmail from '../../../playground/mocks/idp/idx/data/terminal-return-email';
+
+const mock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx')
```

Comment:
probably could just mock `/idp/idx/instrospect` so that it starts with terminal view without going through `identify` page